src/XMakeTasks/Microsoft.Common.CrossTargeting.targets

```diff
@@ -95,19 +95,16 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </PropertyGroup>
   </Target>
 
-  <!--
-    This will import NuGet restore targets, which is a special case separate from the package -> project extension 
-    mechanism below. For obvious reasons,  we need restore to work before any package assets are available.
+  <Import Project="$(CustomAfterMicrosoftCommonCrossTargetingTargets)" Condition="'$(CustomAfterMicrosoftCommonCrossTargetingTargets)' != '' and Exists('$(CustomAfterMicrosoftCommonCrossTargetingTargets)')"/>
```

Comment:
I followed the pattern from common targets and put this before the folder include, but I'm not sure that's right. @Emgarten you mentioned wanting to carry the restore target outside of the MSBuild folder--will it _always_ be outside? If it's in both places, with this ordering the one in `MSBuildExtensionsPath` will win which is the opposite of what we'd want. In that case, I can move this import below the folder one.